### PR TITLE
Easier to update version and added a command.

### PR DESCRIPTION
**Pull request by fillefilip8**
Easier to update version and added a new API function "getVersion()".
And a command :O

### CODE REVIEW

**Comment by StamBoom** [comment]
Maybe also for README.md?

File: src/main/java/us/myles/ViaVersion/api/ViaVersionAPI.java [hunk 1]
```diff
@@ -3,6 +3,11 @@
...
 
 public interface ViaVersionAPI {
-
+    /**
+     * Is player using 1.9?
+     * @param player
+     * @return
+     */
     boolean isPorted(Player player);
+    String getVersion();
```
Comment (Paulomart) [inline src/main/java/us/myles/ViaVersion/api/ViaVersionAPI.java]: Missing whitespace

**Comment by fillefilip8** [comment]
Added more yml info and remove author name from the message in the command.

**Comment by Paulomart** [comment]
You should squash the commits

**Pull request merged**